service/controller.go

```diff
@@ -1241,15 +1285,23 @@ func (s *service) ControllerPublishVolume(
 				err.Error())
 		}
 
-		sdcIPs, err := s.getSDCIPs(nodeID, systemID)
-		if err != nil {
-			return nil, status.Errorf(codes.NotFound, "%s", err.Error())
-		} else if len(sdcIPs) == 0 {
-			return nil, status.Errorf(codes.NotFound, "%s", "received empty sdcIPs")
+		var ipAddresses []string
+
+		ipAddresses, err = s.findNetworkInterfaceIPs()
+		if err != nil || len(ipAddresses) == 0 {
+
```

Comment:
**suggestion:** Might be good to have a log stating that, "no network interfaces founds and we're using/getting the sdc IP"

Reply:
added a comment